subsys/bluetooth/controller/ll_sw/ull.c

```diff
@@ -239,7 +252,8 @@ static struct {
 
 #define LINK_RX_POOL_SIZE (sizeof(memq_link_t) * (RX_CNT + 2 + \
 						  BT_CTLR_MAX_CONN + \
-						  BT_CTLR_ADV_SET))
+						  BT_CTLR_ADV_SET + \
+						  (BT_CTLR_SCAN_SYNC_SET * 2)))
```

Comment:
I'm not sure why we need `BT_CTLR_SCAN_SYNC_SET * 2` rather than just `BT_CTLR_SCAN_SYNC_SET`. Can you elaborate?

Reply:
Ah, that makes sense. Yes please add a comment in the source for this, and then resolve this comment :)